sql/core/src/main/scala/org/apache/spark/sql/streaming/DataStreamReader.scala

```diff
@@ -475,6 +476,23 @@ final class DataStreamReader private[sql](sparkSession: SparkSession) extends Lo
     format("parquet").load(path)
   }
 
+  /**
+   * Define a Streaming DataFrame on a Table. The DataSource corresponding to the table should
+   * support streaming mode.
```

Comment:
If the data source doesn't support, what will happen?

Reply:
It will check in `MicroBatchExecution`/`ContinuousExecution` if the data source doesn't support.